ui/src/app/views/workspace-share.spec.tsx

```diff
@@ -4,27 +4,27 @@ import * as Lolex from 'lolex';
 import * as fp from 'lodash/fp';
 import Select from 'react-select';
 
-import {WorkspaceShare, WorkspaceShareProps, WorkspaceShareState} from './workspace-share';
+import {Props, State, WorkspaceShare} from './workspace-share';
```

Comment:
is it a new convention to not give props/state objects descriptive names, and just use `Props` and `State`?